### DISCUSSION TITLE
Please help to compile windows build with FFMPEG v7.0.2 base on AM 2.7.0

### DISCUSSION THREAD

**Discussion by kent79** [oomek/attractplus#73] [Q&A]
Hello oomek,

I am kent79. This is special reuqest. Could you make some help me as a title since I don't know how to compile its by myself? I find out AM 2.7.0 picture and sound does not sync while video playing. But AM plus 3.0.9 was fixed. I would like to test is it related of FFMPEG. If you have time, please reply me. Thank you so much.

**Comment by oomek** [comment]
I'm sorry. but I only provide support for attractplus.
For any help with baseline attract please follow this link:
https://github.com/mickelson/attract

**Reply by kent79** [reply]
Thank you for quick reply. :)
